### PR TITLE
PS-9250 Release notes for 5.7.44-50 (post-EOL 2)

### PR DESCRIPTION
**Pull request by patrickbirch**
On branch ps-9250
	new file:   docs/release-notes/5.7.44-50.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/release-notes/5.7.44-50.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+Percona Server for MySQL 5.7.44-50 contains the following fixes.
+
+## Bug fixes
+
+* [Bug#36248967]: `mysqldump` sometimes failed to identify the version number correctly.
+
+    * Changed `--source include/have_debug.inc` to `mysql_have_debug.inc`. 
+    
+    * Added a function that cleans the version string by removing invalid characters and generating a warning message.
```
Comment (VarunNagaraju) [inline docs/release-notes/5.7.44-50.md]: These two points are not necessary.

File: docs/release-notes/5.7.44-50.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Percona Server for MySQL 5.7.44-50 (2024-06-19)
+
+This release is part of {{post}}. The fixes are available to [MySQL 5.7 Post-EOL Support from Percona customers]. Community members can [build this release from the source]. We recommend that you upgrade to MySQL 8.0 or you can stay on 5.7; we'll support you either way.
+
+## Release Information
+
+Percona Server for MySQL 5.7.44-50 contains the following fixes.
```
Comment (sleto-it) [inline docs/release-notes/5.7.44-50.md:9]: The word "security" is not mentioned at all in the entire page. This looks strange to me as we are delivering security fixes with this version (part of the EOL program).  I would replace this sentence with:

"Percona Server for MySQL 5.7.44-50 includes all the features and bug fixes available in MySQL 5.7.44 Community Edition in addition to enterprise-grade features and security fixes developed by Percona."

**Review by sleto-it — CHANGES_REQUESTED** [review]

**Comment by VarunNagaraju** [comment]
@patrickbirch We need a third party library advisory section like the last time 

Openssl
======
CVE-2023-6129 
https://www.openssl.org/news/vulnerabilities.html#CVE-2023-6129
OpenSSL versions 1.1.1 and 1.0.2 are not affected by this issue.
If OpenSSL 3.x.y is being used, upgrade to one of the 3.0.13, 3.1.5, 3.2.1 versions.

- Ubuntu https://ubuntu.com/security/notices/USN-6622-1
- Debian https://security-tracker.debian.org/tracker/CVE-2023-6129
- Oracle linux https://linux.oracle.com/errata/ELSA-2024-2447.html
- Red hat https://access.redhat.com/errata/RHSA-2024:2447

Curl
===
CVE-2024-0853
https://curl.se/docs/CVE-2024-0853.html
8.5.0 is the only version affected. Rest of the versions(both higher and lower) are fine.

**Pull request merged**